data-spring-jdbc/src/main/java/org/stellar/anchor/server/sqlite/SQLiteDialect.java

```diff
@@ -0,0 +1,132 @@
+package org.stellar.anchor.server.sqlite;
+
+import java.sql.Types;
+import org.hibernate.dialect.Dialect;
+import org.hibernate.dialect.function.SQLFunctionTemplate;
+import org.hibernate.dialect.function.StandardSQLFunction;
+import org.hibernate.dialect.function.VarArgsSQLFunction;
+import org.hibernate.type.StringType;
+
+public class SQLiteDialect extends Dialect {
```

Comment:
❓ I'm surprised this isn't a default hibernate dialect, why do we have to create this on our own?

Reply:
That’s sad that we have to write the dialect because hibernate does not have the dialect for SQLite.